Update the list of MOOSE apps we know of (#31916)
idaholab/moose: modules/doc/content/application_usage/tracked_apps.md

### Thermal hydraulics and fluid dynamics

- [OpenPronghorn](https://mooseframework.inl.gov/open_pronghorn/) - Coarse mesh computational fluid dynamics
- [barnacle](https://github.com/idaholab/barnacle) - Shallow water equations in MOOSE
- [Saline](https://code.ornl.gov/neams/saline) - Molten salt fluid properties from MSTDB experimental data
- [marlin](https://github.com/idaholab/marlin) - Spectral solver based on MOOSE and GPU-capable

### Mechanics

- [Blackbear](https://github.com/idaholab/blackbear) - Degradation processes in concrete and other structural materials
- [Deer](https://github.com/Argonne-National-Laboratory/deer) - Various mechanics modules for MOOSE
- [MASTODON](https://github.com/idaholab/mastodon) - Structural dynamics, seismic analysis, and risk assessment
- [RACCOON](https://github.com/hugary1995/raccoon) - Massively parallel FEM code for phase-field for fracture

### Materials

- [GRIME](https://github.com/shortlab/grime) - The Grand Radiation Informed Microstructural Evolver: Coupled, spatially dependent radiation damage and microstructural evolution
- [HOGNOSE](https://github.com/shortlab/hognose) - CASL's mesoscale corrosion/oxidation code

> - [Hyrax](https://github.com/UMThorntonGroup/Hyrax) - Zirconium hydride precipitation and growth in LWR nuclear fuel cladding

Reviewer: this is a bit confusing, because non-moose mamba is still being worked. MOOSE version is ancient...

> - [Hyrax](https://github.com/UMThorntonGroup/Hyrax) - Zirconium hydride precipitation and growth in LWR nuclear fuel cladding

Reviewer: <img width="498" height="468" alt="image" src="https://github.com/user-attachments/assets/d2771e3a-9c1a-46b5-8036-3fd651d8bc2e" />

> - [Hyrax](https://github.com/UMThorntonGroup/Hyrax) - Zirconium hydride precipitation and growth in LWR nuclear fuel cladding

Reviewer: I'm assuming I'm going have to rename my code, so didn't mean to be callous towards you. Looking forward to talking in person so I can assure you I'm not trying to be difficult.

- [Redback](https://github.com/pou036/redback) - Rock Mechanics with Dissipative (Thermo-Hydro-Mechanical-Chemical) Feedbacks: joint development between CSIRO and UNSW Australia

### Miscellaneous

- [moopy](https://github.com/aurora-multiphysics/moopy) - Python interface that simplifies (ish) the creation of MOOSE inputs
- [Proteus](https://github.com/aurora-multiphysics/proteus) - For developing Fusion Digital Twins.
- [Ranger](https://github.com/idaholab/ranger) - Python-based auto-response bot that uses the GitHub API and LlamaIndex package to monitor and generate relevant responses for new discussions
- [Virtual Test Bed](https://mooseframework.inl.gov/virtual_test_bed/) - An open repository of simulations of nuclear systems
- [Isopod](https://github.com/idaholab/isopod) - Multiphysics PDE constrained optimization, mostly merged into MOOSE as the optimization module

## Closed Source Applications

These applications require a license. Many of them can be obtained through the [NCRC website](https://ncrcaims.inl.gov/).

### Nuclear Engineering Advanced Modeling and Simulation (NEAMS) tools

- [BISON](https://inlsoftware.inl.gov/product/bison) - Flagship fuels performance code

> - Centipede - A tool to inform engineering scale simulations with atomistic data

Reviewer: Could use a publication link https://www.sciencedirect.com/science/article/pii/S0022311520301884

- [GRIFFIN](https://inlsoftware.inl.gov/product/griffin) - Reactor physics code co-developed between Argonne and Idaho National Laboratory
- [GRIZZLY](https://inlsoftware.inl.gov/product/grizzly) - Nuclear Plant System Degradation Modeling
- [MARMOT](https://inlsoftware.inl.gov/product/marmot) - Nuclear materials phase field and mechanics application
- [PRONGHORN](https://inlsoftware.inl.gov/product/pronghorn) - Transient prismatic and pebble bed reactor analysis code
- [RELAP7](https://inlsoftware.inl.gov/product/relap7) - Next Generation nuclear reactor system safety analysis code (compressible flow)

> - SAM - Advanced nuclear reactor system analysis code (weakly-compressible flow)

Reviewer: https://www.anl.gov/nse/system-analysis-module

- [Sockeye](https://inlsoftware.inl.gov/product/sockeye) - Multi-fidelity studies of heat pipes for nuclear applications

> - SWIFT - A thermo-chemistry tool for metal hydride moderators

Reviewer: Not sure if you do this everywhere, but might want to add "developed at LANL" like you do for Gardensnake.

Reviewer 2: Speaking of this, have you reached a deal on the naming clash? @tophmatthews @dschwen

Reviewer: Tomorrow. Bring your popcorn.

> - SWIFT - A thermo-chemistry tool for metal hydride moderators

Author: ```suggestion
- SWIFT - A thermo-chemistry tool for metal hydride moderators, developed at LANL
```

### Electromagnetics, Plasmas and Additive Manufacturing

- [Apollo](https://github.com/aurora-multiphysics/apollo) - Enabling 3D electromagnetics simulation in MOOSE, using the MFEM FE library.
- [CRANE](https://crane-plasma-chemistry.readthedocs.io/en/latest/) - Chemical ReAction NEtworks for plasma chemistry and thermochemistry problems.
- [Ferret](https://mangerij.github.io/ferret/) - Kernels to implement the time-dependent Landau-Ginzburg theory of phase transitions for simulating ferroelectric materials
- [MALAMUTE](https://mooseframework.inl.gov/malamute/) - Advanced manufacturing modeling and simulation
- [Orpheus](https://github.com/aurora-multiphysics/orpheus) - Additional electromagnetics solvers in MOOSE
- [phaethon](https://github.com/aurora-multiphysics/phaethon) - Fast ion heat fluxes produced by ASCOT5 in MOOSE multiphysics simulations
- [Zapdos](https://github.com/shannon-lab/zapdos) - Low temperature plasma simulation

### Chemistry & species transport

- [DGOSPREY](https://github.com/aladshaw3/dgosprey) - Discontinuous Galerkin Off-gas SeParation and REcoverY model: joint development between INL and GIT

> - [MAMBA](https://github.com/shortlab/mamba) - CASL's code for boiling, heat transfer, and chemistry in porous media

Reviewer: I would move this to "Legacy", and add something like "MOOSE port of the still maintained non-MOOSE MAMBA CASL code" or something like that.

1. The MOOSE developers help keep many applications up to date. But we have to know your application exists to do so!

## Open-source Applications

### Multiphysics frameworks

- [Cardinal](https://github.com/neams-th-coe/cardinal) - Integration of NekRS \& OpenMC with MOOSE for fusion and fission systems high-fidelity simulation
- [SALAMANDER](https://github.com/idaholab/salamander) - Integration of Cardinal with TMAP8 for fusion blanket multiphysics simulations

### Neutronics / Reactor / Particle Physics

- [Aurora](https://aurora-multiphysics.github.io/aurora/) - Integration of OpenMC and MOOSE developed by the UKAEA
- [Frog](https://github.com/idaholab/thermal_to_fusion_converter) - Integration of GEANT-4 for high-energy particle transport
- [Moltres](https://github.com/arfc/moltres) - Molten salt reactor simulation
- [Squirrel](https://github.com/philipJFpfahl/Squirrel) - Point kinetics with delayed neutron precursor advection in molten salt reactors

### Thermal hydraulics and fluid dynamics

- [OpenPronghorn](https://mooseframework.inl.gov/open_pronghorn/) - Coarse mesh computational fluid dynamics

> - [barnacle](https://github.com/idaholab/barnacle) - Shallow water equations in MOOSE

Reviewer 3: I believe that @dschwen has a new shallow water equations implementation (coming)? The last commit to this repo was 6 years ago. I think it can be removed

- [Magpie](https://github.com/idaholab/magpie) - Mesoscale Atomistic Glue Program for Integrated Execution
- [Pika](https://github.com/idaholab/pika) - Phase-field model for micro-structure evolution of ice

### Electromagnetics, Plasmas and Additive Manufacturing

- [Apollo](https://github.com/aurora-multiphysics/apollo) - Enabling 3D electromagnetics simulation in MOOSE, using the MFEM FE library.
- [CRANE](https://crane-plasma-chemistry.readthedocs.io/en/latest/) - Chemical ReAction NEtworks for plasma chemistry and thermochemistry problems.
- [Ferret](https://mangerij.github.io/ferret/) - Kernels to implement the time-dependent Landau-Ginzburg theory of phase transitions for simulating ferroelectric materials
- [MALAMUTE](https://mooseframework.inl.gov/malamute/) - Advanced manufacturing modeling and simulation
- [Orpheus](https://github.com/aurora-multiphysics/orpheus) - Additional electromagnetics solvers in MOOSE
- [phaethon](https://github.com/aurora-multiphysics/phaethon) - Fast ion heat fluxes produced by ASCOT5 in MOOSE multiphysics simulations
- [Zapdos](https://github.com/shannon-lab/zapdos) - Low temperature plasma simulation

### Chemistry & species transport

> - [DGOSPREY](https://github.com/aladshaw3/dgosprey) - Discontinuous Galerkin Off-gas SeParation and REcoverY model: joint development between INL and GIT

Reviewer 3: ```suggestion
- [cats](https://github.com/aladshaw3/cats) - Catalysis And Treatment Simulations (CATS) library for MOOSE
```

- [Proteus](https://github.com/aurora-multiphysics/proteus) - For developing Fusion Digital Twins.
- [Ranger](https://github.com/idaholab/ranger) - Python-based auto-response bot that uses the GitHub API and LlamaIndex package to monitor and generate relevant responses for new discussions
- [Virtual Test Bed](https://mooseframework.inl.gov/virtual_test_bed/) - An open repository of simulations of nuclear systems
- [Isopod](https://github.com/idaholab/isopod) - Multiphysics PDE constrained optimization, mostly merged into MOOSE as the optimization module

## Closed Source Applications

These applications require a license. Many of them can be obtained through the [NCRC website](https://ncrcaims.inl.gov/).

### Nuclear Engineering Advanced Modeling and Simulation (NEAMS) tools

- [BISON](https://inlsoftware.inl.gov/product/bison) - Flagship fuels performance code
- [Centipede](https://www.sciencedirect.com/science/article/pii/S0022311520301884) - A tool to inform engineering scale simulations with atomistic data, developed at LANL
- [GRIFFIN](https://inlsoftware.inl.gov/product/griffin) - Reactor physics code co-developed between Argonne and Idaho National Laboratory

> - [GRIZZLY](https://inlsoftware.inl.gov/product/grizzly) - Nuclear Plant System Degradation Modeling

Reviewer 4: ```suggestion
- [Grizzly](https://inlsoftware.inl.gov/product/grizzly) - Nuclear Plant Component Degradation Modeling
```

- [MARMOT](https://inlsoftware.inl.gov/product/marmot) - Nuclear materials phase field and mechanics application

> - [PRONGHORN](https://inlsoftware.inl.gov/product/pronghorn) - Transient prismatic and pebble bed reactor analysis code

Reviewer 3: Also MSRs?

Author: to my knowledge

- [RELAP7](https://inlsoftware.inl.gov/product/relap7) - Next Generation nuclear reactor system safety analysis code (compressible flow)
- [SAM](https://www.anl.gov/nse/system-analysis-module) - Advanced nuclear reactor system analysis code (weakly-compressible flow), developed at ANL
- [Sockeye](https://inlsoftware.inl.gov/product/sockeye) - Multi-fidelity studies of heat pipes for nuclear applications

> - SWIFT - A thermo-chemistry tool for metal hydride moderators, developed at LANL

Reviewer: yes, thanks to @dschwen for graciously switching.

Combined applications

> - [DIREWOLF](https://www.tandfonline.com/doi/full/10.1080/00295450.2021.1906474) - Advanced reactor multiphysics simulation suite including Griffin, Bison, Sockeye

Reviewer 3: why do we have this since blue crab has sockeye 😦
